asset-transfer-private-data/chaincode-go/collections_config.json

```diff
@@ -0,0 +1,34 @@
+[
+ {
+   "name": "assetCollection",
+   "policy": "OR('Org1MSP.member', 'Org2MSP.member')",
+   "requiredPeerCount": 0,
+   "maxPeerCount": 3,
+   "blockToLive":1000000,
+   "memberOnlyRead": true,
+   "memberOnlyWrite": true
+},
+ {
+   "name": "Org1MSPDetailsCollection",
```

Comment:
for simplicity and easier for new developers, how about renaming these 2 private collection to 
`Org1PrivateCollection & Org2PrivateCollection`

Reply:
Good Idea